src/main/java/com/example/githubProject/ChatApp/Application.java

```diff
@@ -0,0 +1,13 @@
+package com.example.githubProject.ChatApp;
```

Comment:
this package name should be:
`package com.mithril.chatapp.userservice;`

Please refactor the entire project using this package name.